services/channel/service/src/common/setup/register-localization-definitions.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  DeclareEntityDefinitionCommand,
+  DeleteEntityDefinitionCommand,
+  LocalizationServiceMultiTenantMessagingSettings,
+} from '@axinom/mosaic-messages';
+import { StoreOutboxMessage } from '@axinom/mosaic-transactional-inbox-outbox';
+import { ClientBase } from 'pg';
+import {
+  getChannelLocalizationEntityDefinitions,
+  getProgramLocalizationEntityDefinitions,
+} from '../../domains';
+import { Config } from '../config/config-definitions';
+import { requestServiceAccountToken } from '../utils';
+
+const allDefinitions = (config: Config): DeclareEntityDefinitionCommand[] => [
+  ...getChannelLocalizationEntityDefinitions(config.serviceId),
+  ...getProgramLocalizationEntityDefinitions(config.serviceId),
+];
+
+export const registerLocalizationEntityDefinitions = async (
```

Comment:
I would rename the filename to `register-localization-entity-definitions.ts` to align with the function naming.